packages/cli/lib/cli.ts

```diff
@@ -136,7 +128,7 @@ export const generate = async (debug = false, inParentDirectory = false) => {
                 interfaceFileName: TYPES_FILE_NAME.replace('.ts', ''),
                 interfaceNames,
                 mappings,
-                inputs: input && Object.keys(input).length > 0 ? input : '',
+                inputs: input && Object.keys(input).length > 0 ? input.name || input : '',
```

Comment:
why not simply `input?.name || input || ''`

Reply:
Updated with https://github.com/NangoHQ/nango/pull/1501/commits/ec9b187868c8abafa60d7b258f36e4d21b8bcbaa